src/routes/account/+page.server.ts

```
@@ -18,6 +19,16 @@ export const load: PageServerLoad = async ({ locals: { supabase, safeGetSession
 };
 
 export const actions: Actions = {
+	delete: async ({ url, locals: { safeGetSession } }) => {
+		const { session } = await safeGetSession();
+		const baseUrl = url.origin;
+		if (session) {
+			await requestAccountDeletion(session.user.id, session.user.email!, baseUrl);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)